packages/manager/modules/pci/src/projects/project/public-gateways/public-gateways.html

```diff
@@ -66,9 +66,14 @@
         <oui-datagrid-column
             data-title=":: 'pci_projects_project_public_gateway_networks_connected' | translate"
             data-property="interfaces.length"
-            data-type="string"
-            data-filterable
-        ></oui-datagrid-column>
+        >
+            <a
+                data-ng-click="$ctrl.onGoToPrivateNetworkClick($row)"
+                href
+                data-ng-bind="$row.interfaces.length"
+            ></a>
```

Comment:
**nitpick:** Use `a` with `href` or apply link style to `button` with `ng-click`